### PR TITLE
CXP-1308: add a description to highlight the automatic disabling of a catalog

### PR DESCRIPTION
**Pull request by guillaumefer-akeneo**
image
and
image
and
image

### CODE REVIEW

**Review by meganelepalud-akeneo — APPROVED** [review]

File: content/swagger/resources/app_catalogs/routes/app_catalogs_id_product_uuids.yaml [hunk 1]
```diff
@@ -1,7 +1,11 @@
 get:
   summary: Get the list of product uuids
   operationId: "get_app_catalog_product_uuids"
-  description: Coming soon. This endpoint allows you to get the list of uuids of products contained in a catalog.
+  description: |
+    Coming soon. This endpoint allows you to get the list of uuids of products contained in a catalog.
+    Note that a disabled catalog can return an empty list.
+    A catalog can be disabled automatically on the PIM side if a product selection is no longer valid
+    following the deletion of a category or an attribute option.
```
Comment (masacc) [inline content/swagger/resources/app_catalogs/routes/app_catalogs_id_product_uuids.yaml:8]:
```suggestion
    following the deletion of a category or an attribute option for example.
```
As we may have other use cases in the future?

File: content/swagger/resources/app_catalogs/routes/app_catalogs_id_product_uuids.yaml [hunk 2]
```diff
@@ -1,7 +1,11 @@
 get:
   summary: Get the list of product uuids
   operationId: "get_app_catalog_product_uuids"
-  description: Coming soon. This endpoint allows you to get the list of uuids of products contained in a catalog.
+  description: |
+    Coming soon. This endpoint allows you to get the list of uuids of products contained in a catalog.
+    Note that a disabled catalog can return an empty list.
```
Comment (masacc) [inline content/swagger/resources/app_catalogs/routes/app_catalogs_id_product_uuids.yaml:6]: Is it true? I had understood that we will display an error message in that case: "No products to synchronize. The catalog 2983dce8-4129-4fe0-bf8c-8057801e2dd8 has been disabled on the PIM side. Note that you can get catalogs status with the GET /api/rest/v1/catalogs endpoint."

**Review by willy-ahva — COMMENTED** [review]
You should add another "200" tab with the json message exemple.

**Review by masacc — APPROVED** [review]

**Review by RiffVentura — APPROVED** [review]

**Comment by meganelepalud-akeneo** [comment]
managed in another PR

**Pull request closed without merging**